common/src/Microsoft.Azure.IIoT.Hub.Module.Client/src/Default/IoTSdkFactory.cs

```diff
@@ -274,15 +277,27 @@ public sealed class ModuleClientAdapter : IClient {
                 if (IsClosed) {
                     return;
                 }
-                await _client.SendEventAsync(message);
+
+                try {
+                    await _client.SendEventAsync(message);
+                }
+                catch (Exception e) {
+                    _logger.Error(e, "Error sending message to IoT Hub");
```

Comment:
Please undo - we should never eat exceptions in "SDK" like classes, this should be done at the call site (if needed)

Reply:
Done